### DISCUSSION TITLE
libbfd and libiberty

### DISCUSSION THREAD

**Discussion by thoni56** [Homebrew/discussions#2710] [Casks]
Linking with `libbfd` I discovered that to do that I also need `libiberty`, but it is not installed unless you re-install `binutils` with `--enable-install-libiberty`. This is a hazzel for building my package, which is dependent on `libbfd`, under Homebrew.

On many other platforms (Linux) you can link with only `libbfd`. I am not an expert in package managers and dependencies, so I'm not sure how that works.

I'm just wondering if it would be possible to resolve that dependency on Homebrew in the same way, allowing linking only with `libbfd`?

**Comment by SimonKagstrom** [comment]
Old question, but I submitted a PR for this: https://github.com/Homebrew/homebrew-core/pull/168531, since my package also depends on libbfd.

For now, I've added it manually and rebuild binutils for homebrew with `--enable-install-libiberty` as per https://stackoverflow.com/a/65978315 and the instructions here: https://docs.brew.sh/FAQ#can-i-edit-formulae-myself.
